tests/testthat/_snaps/model-offset.md

```diff
@@ -4,13 +4,13 @@
       mold(~ Sepal.Width + offset(Species), iris)
     Condition
       Error in `model_offset()`:
-      ! Column, 'offset(Species)', is tagged as an offset, but is not numeric. All offsets must be numeric.
+      ! Column offset(Species) is tagged as an offset and thus must be numeric, not a <factor> object.
```

Comment:
Maybe it is the specific test case, but I would think that `offset(Species)` should be between back-tics

Reply:
I've moved all column names to a styling via `{.val}`, so that's double quote instead of backticks but treating is as a name worked better in most other contexts.